lib/web/ui/server.go

```diff
@@ -44,26 +44,44 @@ type Server struct {
 	Labels []Label `json:"tags"`
 }
 
+// sortedLabels is a sort wrapper that sorts labels by name
+type sortedLabels []Label
```

Comment:
maybe we should get rid of `Label` type definition and just use `string`. It prevents you from using `sort.Strings()` and you have to create these tiny methods for no reason.

Reply:
in the future, let's keep PR per feature/bug please @alex-kovoy